snapcraft/parts/plugins/_ros.py

```diff
@@ -175,15 +219,33 @@ def _get_stage_runtime_dependencies_commands(self) -> List[str]:
     @overrides
     def get_build_commands(self) -> List[str]:
         return (
-            self._get_workspace_activation_commands()
-            + [
-                "if [ ! -f /etc/ros/rosdep/sources.list.d/20-default.list ]; "
-                "then sudo rosdep init; fi",
+            # pylint: disable=line-too-long
+            [
+                "if [ ! -f /etc/ros/rosdep/sources.list.d/20-default.list ]; then sudo rosdep init; fi",
```

Comment:
We will have to readapt the changes from my MR: https://github.com/snapcore/snapcraft/pull/4176
 to this one for it to work on remote-build